src/react-components/ui-root.js

```diff
@@ -1393,6 +1394,18 @@ class UIRoot extends Component {
       (!this.state.hideLoader || !this.state.didConnectToNetworkedScene) &&
       !(this.props.showSafariDialog || this.props.showWebAssemblyDialog);
 
+    if (isLoading) {
+      console.log("Loading screen shown", {
```

Comment:
I suggest wrapping this in `JSON.parse( JSON.stringify ( ) )` so that you capture the values as they are at this time and don't get fooled in the debugger which will otherwise show what the values are at the time you click to expand the object.

Reply:
See https://github.com/mozilla/hubs/pull/2170